banzai/images.py

```diff
@@ -96,24 +98,25 @@ def writeto(self, filename, fpack=False):
             bpm_hdu = fits.ImageHDU(self.bpm.astype(np.uint8))
             bpm_hdu.name = 'BPM'
             hdu_list.append(bpm_hdu)
+        hdu_list = self.add_astropy_data_tables_to_hdu_list_to_be_saved(hdu_list)
 
-        hdu_list = fits.HDUList(hdu_list)
+        fits_hdu_list = fits.HDUList(hdu_list)
```

Comment:
Why the variable name change?

Reply:
because hdu_list was both used for a list of hdu objects, and as the output of fits.HDUList(list_of_hdus). I thought that was confusing.

Reply:
fits is a file type, so I don't think it is the best variable name here.